template/.polarion/tracker/fields/custom-fields.xml

```diff
@@ -2,60 +2,62 @@
 <!-- SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
 SPDX-License-Identifier: Apache-2.0 -->
 <fields>
```

Comment:
Reformating: Ordering the attributes. Only **classes** and **classes_reverse** are new.